stix_shifter_modules/guardium/stix_translation/json/to_stix_map.json

```diff
@@ -152,5 +159,91 @@
       "transformer": "GuardiumToTimestamp",
       "cybox": false
     }
-  ]
+  ],
+  "Url": {
+    "key": "url.value"
+  },
+  "FileName": {
+    "key": "file.name"
+  },
+  "Payload": {
+    "key": "artifact.payload_bin"
+  },
+  "DestinationPort": {
+    "key": "network-traffic.dst_port",
+    "object": "nt",
+    "transformer": "ToInteger"
+  },
+  "SourcePort": {
+    "key": "network-traffic.src_port",
+    "object": "nt",
+    "transformer": "ToInteger"
+  },
+  "NetworkProtocol": {
+    "key": "network-traffic.protocols",
+    "object": "nt",
+    "transformer": "ToLowercaseArray"
+  },
+  "DomainName": {
+    "key": "domain-name.value"
+  },
+  "Process": {
+    "Path": [
+      {
+        "object": "proc",
+        "key": "process.command_line"
+      }
+    ],
+    "Pid": [
+      {
+        "object": "proc",
+        "key": "process.id"
+      }
+    ]
+  },
+  "violation":{
+	"key":"x-ibm-finding.name",
+        "object":"finding"
+  },
+ "Threat Category":{
+	"key":"x-ibm-finding.name",
+        "object":"finding"
+  },
+  "details":{
+	"key":"x-ibm-finding.details",
+        "object":"finding"
+  },
+ "database":{
+	"key":"x-ibm-finding.dst_device",
+        "object":"finding"
+  },
+ "Database":{
+	"key":"x-ibm-finding.dst_device",
+        "object":"finding"
+  },
+  "severity":[
+     {
+	"key":"x-ibm-guardium.severity",
+     "transformer": "GuardiumMapSeverity"},
+     {
+    "key":"x-ibm-finding.finding_type",
+	"object":"finding",
+    "transformer": "GuardiumQS"}
+  ],
+  "Severity":[
+     {
+	"key":"x-ibm-guardium.severity"},
```

Comment:
rename this to just `x-guardium`.

Reply:
it gives me a warning if it is only x-guardium:
`Warning: observed-data--a1138ba2-36c9-49a8-bf69-f958f34cb1bb: {101} Custom Observable Object type 'x-guardium' should start with 'x-' followed by a source unique identifier (like a domain name with dots replaced by hyphens), a hyphen and then the name.`

Reply:
I changed it anyway.